guides/common/modules/ref_storage-guidelines.adoc

```diff
@@ -9,7 +9,7 @@ This is a requirement for the `puppetserver` service to work.
 
 * Because most {ProductName} data is stored in the `/var` directory, mounting `/var` on LVM storage can help the system to scale.
 
-* Use high-bandwidth, low-latency storage for the `/var/lib/pulp/` directories.
+* Use high-bandwidth, low-latency storage for the `/var/lib/pulp/` and PostgreSQL `/var/lib/pgsql/` directories.
 As {ProjectName} has many operations that are I/O intensive, using high latency, low-bandwidth storage causes performance degradation.
 Ensure your installation has a speed in the range 60{range}80 Megabytes per second.
```

Comment:
I think this is outdated now. Modern storage is easily able to do 100x this (current fastest NVME single drivers are in the 10 GB/s+ range so getting to the 200x). I think the high-bandwidth and low latency should cover it already
```suggestion
```